eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -117,3 +117,106 @@ function Process-ReviewStatusCode($statusCode, $packageName, $apiApprovalStatus,
   $packageNameStatus.IsApproved = $packageNameApproved
   $packageNameStatus.Details = $packageNameApprovalDetails
 }
+
+function Set-ApiViewCommentForRelatedIssues {
```

Comment:
Where is this used?

Reply:
This is called by the GitHub workflow https://github.com/Azure/azure-rest-api-specs/pull/33144